product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx

```diff
@@ -11,48 +11,48 @@ You must configure the PEM database server to work with PgBouncer. This example
 
 1.  Create a dedicated user named pgbouncer on the PEM database server:
 
-    ```ini
-    pem=# CREATE USER pgbouncer PASSWORD 'ANY_PASSWORD' LOGIN;
+    ```sql
+    CREATE USER pgbouncer PASSWORD 'ANY_PASSWORD' LOGIN;
     CREATE ROLE
```

Comment:
Create role is output and should be preceded by __OUTPUT__ in the code block to style it correctly.

Reply:
got it 👍